### PR TITLE
Support of X230 board with FHD mod

### PR DESCRIPTION
**Pull request by jyvet**
The strategy would be the following in order to make a release: find the latest coreboot commit on which the latest version of the FHD patch (https://review.coreboot.org/c/coreboot/+/28950) can be applied. Allowing to cherry-pick the patch makes the future releases more flexible as it may apply without conflict on a newer version of coreboot.

The first patch of this serie allow to cherry-pick a patch which is hosted on the gerrit owned by coreboot.org.
The second patch creates a x230_fhd directories. f5b3a49b76 commit is used as the FHD patch does not apply on 24e6221706 (coreboot commit used by skulls-x230-0.1.10).

### CODE REVIEW

**Comment by jyvet** [comment]
I'll test the generated binaries (free & nonfree) after a first round of reviews. My setup: nitrocaster mod + LG LP125WF4-SPB1.

**Comment by jyvet** [comment]
I've just tested the binaries. There is a tiny issue I'm not sure to be able to fix: The splashscreen is displayed but when pressing ESC, the screen goes black and I'm not able to see the menu with the different payloads. I'm able to control the backlight and the screen is turned off when the laptop is closed. 

Any idea how to fix the boot menu?

**Comment by merge** [comment]
doesn't cherry-picking break reproducibility? I think we need to know the final git HEAD we build which won't be possible with this.

**Comment by jyvet** [comment]
I'm sorry, I'm not sure to understand. With this PR, skulls would apply the FHD patch on the top of a known git HEAD. It does not checkout the version of the FHD patch on which it was originally rebased on. 

To me a release would be fully reproducible. Assuming: 

-  coreboot version is based on the commit hash used contained in the name of the config files (e.g.  x230_fhd/free-defconfig-f5b3a49b76 and x230_fhd/nonfree-defconfig-f5b3a49b76).

- x230_fhd/build.sh contains the patch set revision to apply (e.g. revision 12 with --patchset refs/changes/50/28950/12 x230_fhd).   

From previous examples, if the refs/changes/50/28950/12 x230_fhd patch set applies on the top of f5b3a49b76, then the version can be compiled, tested and released (and reproducibility is ensured).

So basically when preparing a new release, we would need to ensure the patch set applies (revision 12 in my example) on the top of the new git HEAD. If it does not apply, we can modify x230_fhd/build.sh to use a newer revision of the FHD patch set (revision 13 would be  --patchset refs/changes/50/28950/13 x230_fhd).
